repose-aggregator/components/filters/uri-user-filter/build.gradle

```diff
@@ -1,12 +1,15 @@
 dependencies {
     compile project(":repose-aggregator/core/core-service-api")
     compile project(":repose-aggregator/commons/commons-utilities")
-    compile project(":repose-aggregator/core/core-lib") //todo: remove this once the wrappers have been removed
     compile "org.slf4j:slf4j-api"
+    compile "org.scala-lang:scala-library"
+    compile "com.typesafe.scala-logging:scala-logging-slf4j_2.10"
 
     provided "javax.servlet:javax.servlet-api"
     provided "javax.inject:javax.inject"
 
     testCompile "junit:junit"
+    testCompile "org.scalatest:scalatest_2.10"
     testCompile "org.mockito:mockito-all"
+    testCompile "org.springframework:spring-test"
```

Comment:
it hangs out several places, that aren't strictly necessary. I suspect it was copy pasta in the maven builds. it was not needed in the gradle build at the time I converted it, but things might have changed since then. Or somebody was having a problem and decided to include it to get better output from their tests for debugging porpoises